### PR TITLE
uv integration

### PR DESCRIPTION
**Pull request by julienp**
Add a blog post about our new uv integration.

The video will be released on the 27th November, merge this once the video is public & released.

### CODE REVIEW

File: content/blog/python-uv-toolchain/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    virtualenv: .venv
+```
+
+You can find more information on how to use uv in Pulumi in our [Python documentation](/docs/iac/languages-sdks/python/#uv).
+
+## What Makes uv so Fast?
+
+If you are  curious about how uv is so fast, listen in on this discussion between [Adam Gordon Bell](/blog/author/adam-gordon-bell/) and [Julien Poissonnier](/blog/author/julien-poissonnier/) as they discuss uv:
+
+{{< youtube "hxi7ZL9H0IU?rel=0" >}}
```
Comment (julienp) [inline content/blog/python-uv-toolchain/index.md:41]: The video will be released on the 27th

File: content/blog/python-uv-toolchain/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Pulumi + uv: Fast Python Package and Project Management"
+date: 2024-11-27T12:43:45+01:00
+draft: false
+meta_desc: Uv is an extremely fast Python package and project manager that is now fully integrated into Pulumi
```
Comment (ericrudder) [inline content/blog/python-uv-toolchain/index.md:5]:
```suggestion
meta_desc: uv is an extremely fast Python package and project manager that is now fully integrated into Pulumi
```

(source, kd lang .. and https://repository.rit.edu/cgi/viewcontent.cgi?article=3057&context=article#:~:text=As%20for%20%E2%80%9Ca%20proper%20noun,APA%2C%202020:%20174)

Comment (julienp) [reply content/blog/python-uv-toolchain/index.md:5]: Amazing

Comment (julienp) [reply content/blog/python-uv-toolchain/index.md]: I honestly struggled with this and reworded some things to not have "uv" at the beginning in some places 😬

**Review by adamgordonbell — APPROVED** [review]

**Pull request merged**